Code/wwnet/network-typedefs.h

```diff
@@ -0,0 +1,13 @@
+#ifndef NETWORKTYPEDEFS_H
+#define NETWORKTYPEDEFS_H
+
+#ifdef _WINDOWS
+#include <winsock.h>
+#else
+#include <arpa/inet.h>
+typedef struct sockaddr_in SOCKADDR_IN;
+typedef int SOCKET;
+typedef SOCKADDR_IN* LPSOCKADDR_IN;
```

Comment:
I personally hate these windowsisms and would rather just replace their use in the code with the standard `struct sockaddr_in`. I'd only be inclined to keep SOCKETS since its type really is different between platforms.